src/kinvey.ts

```diff
@@ -0,0 +1,45 @@
+import { Promise } from 'es6-promise';
+import url from 'url';
+import {
+  Kinvey as CoreKinvey,
+  isDefined,
+  KinveyError,
+  CacheRequest,
+  RequestMethod
+} from 'kinvey-js-sdk/dist/export';
+import { Client } from './client';
+
+const USERS_NAMESPACE = 'user';
+const ACTIVE_USER_COLLECTION_NAME = 'kinvey_active_user';
+
+interface ClientConfig {
```

Comment:
Ok, you'll need to modify the `kinvey.d.ts` as well. 
As an alternative, what about adding a `ClientConfig` definition in `kinvey.d.ts`?